physics/equipotential_body.hpp

```diff
@@ -178,14 +178,151 @@ auto Equipotential<InertialFrame, Frame>::ComputeLines(
       states.push_back(State{});
       continue;
     }
-
+    LOG(ERROR) << total_energy << " "
+               << dynamic_frame_->GeometricPotential(
+                      t, equipotential_position.value());
     // Compute that equipotential.
     states.push_back(ComputeLine(plane, t, equipotential_position.value()));
   }
 
   return states;
 }
 
+template<typename InertialFrame, typename Frame>
+auto Equipotential<InertialFrame, Frame>::ComputeLines(
+    Plane<Frame> const& plane,
+    Instant const& t,
+    std::vector<Position<Frame>> const& peaks,
+    std::vector<Well> const& wells,
+    std::function<Position<Frame>(Position<Frame>)> towards_infinity,
+    SpecificEnergy const& energy) const -> std::vector<State> {
+  using WellIterator = typename std::vector<Well>::const_iterator;
+  LOG(ERROR) << "V=" << energy;
```

Comment:
Do we need to log the parameter?

Reply:
Yes, otherwise the other logs are hard to interpret.